src/Translatable.php

```diff
@@ -105,6 +109,10 @@ protected function getTranslation(string $locale)
             return $this->translationCache[$locale];
         }
 
+        if (!$this->exists) {
+            return null;
```

Comment:
StyleCI whines about this, but we need to return `null` at this point. Suggestions?

Reply:
@vinkla

Reply:
`return;` should be the equivalent of `return null;`

Reply:
Okay, don't worry. I'll go over the syntax.